lib/drivers/tandemDriver.js

```diff
@@ -1009,7 +1000,14 @@ module.exports = function (config) {
       PUMP_LOG_RECORDS.LID_BOLUS_REQUESTED_MSG1.value,
       PUMP_LOG_RECORDS.LID_BOLUS_REQUESTED_MSG2.value,
       PUMP_LOG_RECORDS.LID_BOLUS_REQUESTED_MSG3.value,
-      PUMP_LOG_RECORDS.LID_IDP_TD_SEG.value
+      PUMP_LOG_RECORDS.LID_IDP_TD_SEG.value,
```

Comment:
Since we're not reading settings history, it seems odd to me to include `LID_IDP_TD_SEG`, but it's (very) possible we discussed this, and I've forgot. Remind me?